phy/cluster/manual/default_settings.py

```diff
@@ -66,10 +72,28 @@
 }
 
 
+gui_config = [
+    ('wizard', {'position': 'right'}),
+    ('features', {'position': 'left'}),
+    ('correlograms', {'position': 'left'}),
+    ('waveforms', {'position': 'right'}),
+    ('traces', {'position': 'right'}),
+]
+
+
 # -----------------------------------------------------------------------------
-# Internal settings
+# Store settings
 # -----------------------------------------------------------------------------
 
 # Number of spikes to load at once from the features_masks array
 # during the cluster store generation.
-manual_clustering.store_chunk_size = 100000
+store_chunk_size = 100000
+
+
+# -----------------------------------------------------------------------------
+# Internal settings
+# -----------------------------------------------------------------------------
+
+waveforms_scale_factor = .01
+features_scale_factor = .01
```

Comment:
it would be good to add a comment stating which scale this is (height? width of view? channel height?)